### ISSUE TITLE
Rector 0.15.5 need to be released as PHPStan 1.9.10+ cause circular reference error

### ISSUE THREAD

**Issue by samsonasik** [rectorphp/rector#7715]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | e.g. v0.15.4 (invoke `vendor/bin/rector --version`)             |

## Minimal PHP Code Causing Issue

Current Rector 0.15.4 with update using PHPStan 1.9.11 cause circular reference:

```
/vendor/rector/rector/vendor/phpstan/phpstan/phpstan.phar/vendor/nette/di/src/DI/Container.php on line 156
Fatal error: Uncaught _PHPStan_503e82092\Nette\InvalidStateException: Circular reference detected for services: betterReflectionSourceLocator, 0281, 0275, 080, 079, fileFinderScan, fileExcluderScan, 0302, originalBetterReflectionReflector. in phar:///vendor/rector/rector/vendor/phpstan/phpstan/phpstan.phar/vendor/nette/di/src/DI/Container.php:156
```

## Expected Behaviour

It should keep working, the `dev-main` already resolved it. To make it working with phpstan 1.9.10+, new release is needed.

**Comment by samsonasik** [comment]
@TomasVotruba thank you, it resolved in latest release 0.15.5 https://github.com/rectorphp/rector/releases/tag/0.15.5

**Comment by TomasVotruba** [comment]
Done :+1:
